_posts/2021-06-30-join-demodam-hackathon.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Join the Demodam hackathon"
+date: 2021-06-30
+author: Alba Roza
+type: blogpost
+excerpt: Information on how to join the following Demodam hackathon
+categories:
+  - Demodam
+---
+
+# Join the Demodam hackathon
+
+Demodam is a new concept that brings together a community of developers, companies, municipalities and users. The idea of Demodam, a demo of a municipal website, enables reusable open source products and services available for municipalities in a single market place.
+
+## The hackathon
+
+During the hackathon of July 5-9 Demodam will take shape - hopefully supported by a grant from the Dutch Ministry of the Interior. Would you like to contribute to this interesting project? Here are the details you need to know.
+
+In [this collaborative document](https://docs.google.com/presentation/d/1ueLpnCIA06f05uuSTop8DU9olLb0M34iR_HrjsebRfI/edit#slide=id.p) you can find all the information about it. As in other [Common Ground](https://commonground.nl/cms/view/77953921-4161-462c-af24-6ac3cd7d61a7/common-ground-wat-hoe-en-waarom) hackathons, you just have to go to slide 17 and add your team name.
```

Comment:
This sounds strange (and perhaps it is!). Do you always have to go to slide 17 in Common Ground hackathons?

Reply:
Not _that_ specific slide but it's the thing they've done as far as I know

Reply:
Suggest removing the Common Ground link here - it muddies which link you need to follow to sign up. You could instead introduce CG support in the intro, including the link, if you think it's important to mention that context.